hapi-fees/src/test/java/com/hedera/services/usage/schedule/ScheduleCreateUsageTest.java

```diff
@@ -56,6 +56,7 @@
 
 	Key adminKey = KeyUtils.A_THRESHOLD_KEY;
```

Comment:
We should add new test case for `createsExpectedDeltaForMemo` and have new private function `givenOpWithMemo` that created scheduled TX operation with memo populated in it